Category:  coch
Dataset:  all_sw_coch_preprocessed_v2_param_v39_augm_v0_cleaned_8000.pkl
File:  conv.train2240
Description:  augm
2021-06-22 23:07:54.998556: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
WARNING:tensorflow:From /home/alirachidi/classification_algorithm/trainers/main/models/conv/modules/augmentation.py:19: experimental_run_functions_eagerly (from tensorflow.python.eager.def_function) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.config.run_functions_eagerly` instead of the experimental version.
2021-06-22 23:07:57.122558: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-06-22 23:07:57.123451: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcuda.so.1
2021-06-22 23:07:57.170014: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:07:57.170735: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:00:04.0 name: Tesla T4 computeCapability: 7.5
coreClock: 1.59GHz coreCount: 40 deviceMemorySize: 14.75GiB deviceMemoryBandwidth: 298.08GiB/s
2021-06-22 23:07:57.170764: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
2021-06-22 23:07:57.174180: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.11
2021-06-22 23:07:57.174273: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.11
2021-06-22 23:07:57.175334: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcufft.so.10
2021-06-22 23:07:57.175624: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcurand.so.10
2021-06-22 23:07:57.176610: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusolver.so.10
2021-06-22 23:07:57.177512: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusparse.so.11
2021-06-22 23:07:57.177678: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.8
2021-06-22 23:07:57.177783: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:07:57.178561: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:07:57.179294: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
Tensorflow Version: 2.4.0
Num GPUs Available:  1
-----------------------
Using module: nv/train2240.py
Using train_file: ../../data/datasets/all_sw_coch_preprocessed_v2_param_v39_augm_v0_cleaned_8000.pkl
Job directory: ../../new_cache/conv__2240__06_2307__v39__augm
Parameters: {'N_CLASSES': 1, 'SR': 8000, 'BATCH_SIZE': 16, 'LR': 0.001, 'SHAPE': [128, 1250], 'LL2_REG': 0, 'WEIGHT_DECAY': 0.001, 'N_EPOCHS': 65, 'FACTOR': 0.5, 'PATIENCE': 6, 'MIN_LR': 1e-07, 'LABEL_SMOOTHING': 0.4, 'EPSILON': 1e-07, 'ES_PATIENCE': 20, 'OVERSAMPLE': 0, 'MIN_DELTA': 0, 'PARAM': 'v39', 'EPOCH_START': 1, 'JORDAN_DATASET': 1, 'CLAUSE': 0, 'CUBEROOTING': 1, 'NORMALIZING': 1, 'AUGMENTATION': 0, 'TARGET': 1, 'INITIAL_CHANNELS': 1, 'ADAPTIVE_LR': 1, 'CLASS_WEIGHTS': 1, 'SIX': 0, 'CONCAT': 0, 'TESTING': 0}
Augmentation parameters for audio: {'ADD': 1, 'QUANTITY': 300, 'WAV_PATH': 'v42'}
Augmentation parameters for spectrograms: {'ADD': 0, 'QUANTITY': 300, 'TIME_MASKING': 1, 'FREQUENCY_MASKING': 1, 'LOUDNESS': 0}
-----------------------
Collecting Variables...
All variables have been collected.
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow_addons/utils/ensure_tf_install.py:68: UserWarning: Tensorflow Addons supports using Python ops for all Tensorflow versions above or equal to 2.2.0 and strictly below 2.3.0 (nightly versions are not supported). 
 The versions of TensorFlow you are currently using is 2.4.0 and is not supported. 
Some things might work, some things might not.
If you were to encounter a bug, do not file an issue.
If you want to make sure you're using a tested and supported configuration, either change the TensorFlow version or the TensorFlow Addons's version. 
You can find the compatibility matrix in TensorFlow Addon's readme:
https://github.com/tensorflow/addons
  UserWarning,
wandb: W&B syncing is set to `offline` in this directory.  Run `wandb online` or set WANDB_MODE=online to enable cloud syncing.
2021-06-22 23:08:01.744847: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-06-22 23:08:01.745068: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:08:01.745789: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:00:04.0 name: Tesla T4 computeCapability: 7.5
coreClock: 1.59GHz coreCount: 40 deviceMemorySize: 14.75GiB deviceMemoryBandwidth: 298.08GiB/s
2021-06-22 23:08:01.745831: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
2021-06-22 23:08:01.745883: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.11
2021-06-22 23:08:01.745924: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.11
2021-06-22 23:08:01.745947: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcufft.so.10
2021-06-22 23:08:01.745970: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcurand.so.10
2021-06-22 23:08:01.746004: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusolver.so.10
2021-06-22 23:08:01.746025: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusparse.so.11
2021-06-22 23:08:01.746045: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.8
2021-06-22 23:08:01.746129: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:08:01.746821: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:08:01.747450: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
2021-06-22 23:08:01.747489: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
2021-06-22 23:08:02.426397: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1261] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-06-22 23:08:02.426443: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1267]      0 
2021-06-22 23:08:02.426451: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1280] 0:   N 
2021-06-22 23:08:02.426727: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:08:02.427513: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:08:02.428241: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:941] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-06-22 23:08:02.428967: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1406] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 13968 MB memory) -> physical GPU (device: 0, name: Tesla T4, pci bus id: 0000:00:04.0, compute capability: 7.5)
Pneumonia patients: 148, Non-pneumonia patients: 2873
Pneumonia patients: 51, Non-pneumonia patients: 0
-----------------------
Number of train recordings: 2457 with pneumonia: 157 and non-pneumonia: 2300
Number of val recordings: 615 with pneumonia: 42 and non-pneumonia: 573
-----------------------
Initializing weights...
weights = {0: 0.5257570483814827, 1: 10.20608108108108}
Model: "conv2d"
______________________________________________________________________________________________________________
Layer (type)                        Output Shape            Param #      Connected to                         
==============================================================================================================
input_1 (InputLayer)                [(None, 128, 1250, 1)]  0                                                 
______________________________________________________________________________________________________________
batch_normalization (BatchNormaliza (None, 128, 1250, 1)    4            input_1[0][0]                        
______________________________________________________________________________________________________________
conv2d (Conv2D)                     (None, 128, 1250, 16)   32           batch_normalization[0][0]            
______________________________________________________________________________________________________________
conv2d_2 (Conv2D)                   (None, 128, 1250, 16)   32           batch_normalization[0][0]            
______________________________________________________________________________________________________________
max_pooling2d (MaxPooling2D)        (None, 128, 1250, 1)    0            batch_normalization[0][0]            
______________________________________________________________________________________________________________
conv2d_1 (Conv2D)                   (None, 128, 1250, 16)   2320         conv2d[0][0]                         
______________________________________________________________________________________________________________
conv2d_3 (Conv2D)                   (None, 128, 1250, 16)   6416         conv2d_2[0][0]                       
______________________________________________________________________________________________________________
conv2d_4 (Conv2D)                   (None, 128, 1250, 16)   32           max_pooling2d[0][0]                  
______________________________________________________________________________________________________________
concatenate (Concatenate)           (None, 128, 1250, 48)   0            conv2d_1[0][0]                       
                                                                         conv2d_3[0][0]                       
                                                                         conv2d_4[0][0]                       
______________________________________________________________________________________________________________
average_pooling2d (AveragePooling2D (None, 64, 625, 48)     0            concatenate[0][0]                    
______________________________________________________________________________________________________________
batch_normalization_1 (BatchNormali (None, 64, 625, 48)     192          average_pooling2d[0][0]              
______________________________________________________________________________________________________________
dropout (Dropout)                   (None, 64, 625, 48)     0            batch_normalization_1[0][0]          
______________________________________________________________________________________________________________
inverted_residual (InvertedResidual (None, 64, 625, 32)     35840        dropout[0][0]                        
______________________________________________________________________________________________________________
inverted_residual_1 (InvertedResidu (None, 64, 625, 32)     20864        inverted_residual[0][0]              
______________________________________________________________________________________________________________
average_pooling2d_1 (AveragePooling (None, 32, 312, 32)     0            inverted_residual_1[0][0]            
______________________________________________________________________________________________________________
batch_normalization_2 (BatchNormali (None, 32, 312, 32)     128          average_pooling2d_1[0][0]            
______________________________________________________________________________________________________________
dropout_1 (Dropout)                 (None, 32, 312, 32)     0            batch_normalization_2[0][0]          
______________________________________________________________________________________________________________
inverted_residual_2 (InvertedResidu (None, 32, 312, 64)     27136        dropout_1[0][0]                      
______________________________________________________________________________________________________________
inverted_residual_3 (InvertedResidu (None, 32, 312, 64)     66304        inverted_residual_2[0][0]            
______________________________________________________________________________________________________________
average_pooling2d_2 (AveragePooling (None, 16, 156, 64)     0            inverted_residual_3[0][0]            
______________________________________________________________________________________________________________
batch_normalization_3 (BatchNormali (None, 16, 156, 64)     256          average_pooling2d_2[0][0]            
______________________________________________________________________________________________________________
dropout_2 (Dropout)                 (None, 16, 156, 64)     0            batch_normalization_3[0][0]          
______________________________________________________________________________________________________________
inverted_residual_4 (InvertedResidu (None, 16, 156, 128)    91136        dropout_2[0][0]                      
______________________________________________________________________________________________________________
inverted_residual_5 (InvertedResidu (None, 16, 156, 128)    230912       inverted_residual_4[0][0]            
______________________________________________________________________________________________________________
average_pooling2d_3 (AveragePooling (None, 8, 78, 128)      0            inverted_residual_5[0][0]            
______________________________________________________________________________________________________________
batch_normalization_4 (BatchNormali (None, 8, 78, 128)      512          average_pooling2d_3[0][0]            
______________________________________________________________________________________________________________
dropout_3 (Dropout)                 (None, 8, 78, 128)      0            batch_normalization_4[0][0]          
______________________________________________________________________________________________________________
inverted_residual_6 (InvertedResidu (None, 8, 78, 256)      329728       dropout_3[0][0]                      
______________________________________________________________________________________________________________
inverted_residual_7 (InvertedResidu (None, 8, 78, 256)      855040       inverted_residual_6[0][0]            
______________________________________________________________________________________________________________
average_pooling2d_4 (AveragePooling (None, 4, 39, 256)      0            inverted_residual_7[0][0]            
______________________________________________________________________________________________________________
batch_normalization_5 (BatchNormali (None, 4, 39, 256)      1024         average_pooling2d_4[0][0]            
______________________________________________________________________________________________________________
dropout_4 (Dropout)                 (None, 4, 39, 256)      0            batch_normalization_5[0][0]          
______________________________________________________________________________________________________________
inverted_residual_8 (InvertedResidu (None, 4, 39, 512)      1249280      dropout_4[0][0]                      
______________________________________________________________________________________________________________
inverted_residual_9 (InvertedResidu (None, 4, 39, 512)      3282944      inverted_residual_8[0][0]            
______________________________________________________________________________________________________________
global_average_pooling2d (GlobalAve (None, 512)             0            inverted_residual_9[0][0]            
______________________________________________________________________________________________________________
dense (Dense)                       (None, 1)               513          global_average_pooling2d[0][0]       
==============================================================================================================
Total params: 6,200,645
Trainable params: 6,159,139
Non-trainable params: 41,506
______________________________________________________________________________________________________________
2021-06-22 23:08:03.696626: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
2021-06-22 23:08:03.699031: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 2299995000 Hz
2021-06-22 23:08:04.061971: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.8
2021-06-22 23:08:05.800450: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.11
2021-06-22 23:08:06.255672: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.11
Epoch 1/65
154/154 - 231s - loss: 1.0910 - accuracy: 0.5902 - val_loss: 0.7184 - val_accuracy: 0.9317
Confusion matrix: 
 [[573   0]
 [ 42   0]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1272: UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 due to no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
Normalized Confusion matrix: 
 [[1. 0.]
 [1. 0.]]
Validation f1: 0.00
Validation accuracy: 93.17
Validation recall: 0.00
Validation precision: 0.00
Validation AUC: 0.07405292258380519
Validation class accuracies: [1.0, 0.0]
The validation tracker metric at 0 hasn't increased by 0 in 1 epochs
Epoch 2/65
154/154 - 234s - loss: 0.6963 - accuracy: 0.7766 - val_loss: 0.5640 - val_accuracy: 0.9317
Confusion matrix: 
 [[573   0]
 [ 42   0]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1272: UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 due to no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
Normalized Confusion matrix: 
 [[1. 0.]
 [1. 0.]]
Validation f1: 0.00
Validation accuracy: 93.17
Validation recall: 0.00
Validation precision: 0.00
Validation AUC: 0.07848601890647212
Validation class accuracies: [1.0, 0.0]
The validation tracker metric at 0 hasn't increased by 0 in 2 epochs
Epoch 3/65
154/154 - 235s - loss: 0.7068 - accuracy: 0.7538 - val_loss: 0.5527 - val_accuracy: 0.9317
Confusion matrix: 
 [[573   0]
 [ 42   0]]
Normalized Confusion matrix: 
 [[1. 0.]
 [1. 0.]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1272: UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 due to no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
Validation f1: 0.00
Validation accuracy: 93.17
Validation recall: 0.00
Validation precision: 0.00
Validation AUC: 0.07794048811855775
Validation class accuracies: [1.0, 0.0]
The validation tracker metric at 0 hasn't increased by 0 in 3 epochs
Epoch 4/65
154/154 - 235s - loss: 0.6850 - accuracy: 0.7998 - val_loss: 0.6584 - val_accuracy: 0.7496
Confusion matrix: 
 [[432 141]
 [ 13  29]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.754 0.246]
 [0.31  0.69 ]]
Validation f1: 27.36
Validation accuracy: 74.96
Validation recall: 69.05
Validation precision: 17.06
Validation AUC: 0.3633904630850081
Validation class accuracies: [0.7539267015706806, 0.6904761904761905]
Saving model...
Epoch 5/65
154/154 - 235s - loss: 0.6795 - accuracy: 0.8116 - val_loss: 0.7513 - val_accuracy: 0.6992
Confusion matrix: 
 [[389 184]
 [  1  41]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.679 0.321]
 [0.024 0.976]]
Validation f1: 30.71
Validation accuracy: 69.92
Validation recall: 97.62
Validation precision: 18.22
Validation AUC: 0.40593432320820216
Validation class accuracies: [0.6788830715532286, 0.9761904761904762]
Saving model...
Epoch 6/65
154/154 - 235s - loss: 0.6664 - accuracy: 0.8274 - val_loss: 0.6058 - val_accuracy: 0.8488
Confusion matrix: 
 [[484  89]
 [  4  38]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.845 0.155]
 [0.095 0.905]]
Validation f1: 44.97
Validation accuracy: 84.88
Validation recall: 90.48
Validation precision: 29.92
Validation AUC: 0.5505955325905181
Validation class accuracies: [0.8446771378708552, 0.9047619047619048]
Saving model...
Epoch 7/65
154/154 - 235s - loss: 0.6681 - accuracy: 0.8409 - val_loss: 0.9083 - val_accuracy: 0.5220
Confusion matrix: 
 [[281 292]
 [  2  40]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.49  0.51 ]
 [0.048 0.952]]
Validation f1: 21.39
Validation accuracy: 52.20
Validation recall: 95.24
Validation precision: 12.05
Validation AUC: 0.3447786040667596
Validation class accuracies: [0.49040139616055844, 0.9523809523809523]
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 1 epochs
Epoch 8/65
154/154 - 235s - loss: 0.6663 - accuracy: 0.8197 - val_loss: 0.6292 - val_accuracy: 0.7870
Confusion matrix: 
 [[445 128]
 [  3  39]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.777 0.223]
 [0.071 0.929]]
Validation f1: 37.32
Validation accuracy: 78.70
Validation recall: 92.86
Validation precision: 23.35
Validation AUC: 0.4549537607260894
Validation class accuracies: [0.7766143106457243, 0.9285714285714286]
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 2 epochs
Epoch 9/65
154/154 - 235s - loss: 0.6514 - accuracy: 0.8575 - val_loss: 0.6506 - val_accuracy: 0.7967
Confusion matrix: 
 [[452 121]
 [  4  38]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.789 0.211]
 [0.095 0.905]]
Validation f1: 37.81
Validation accuracy: 79.67
Validation recall: 90.48
Validation precision: 23.90
Validation AUC: 0.4670462714635642
Validation class accuracies: [0.7888307155322862, 0.9047619047619048]
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 3 epochs
Epoch 10/65
154/154 - 235s - loss: 0.6494 - accuracy: 0.8718 - val_loss: 0.7335 - val_accuracy: 0.5724
Confusion matrix: 
 [[311 262]
 [  1  41]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.543 0.457]
 [0.024 0.976]]
Validation f1: 23.77
Validation accuracy: 57.24
Validation recall: 97.62
Validation precision: 13.53
Validation AUC: 0.5909267862390262
Validation class accuracies: [0.5427574171029669, 0.9761904761904762]
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 4 epochs
Epoch 11/65
154/154 - 236s - loss: 0.6462 - accuracy: 0.8681 - val_loss: 0.6515 - val_accuracy: 0.7935
Confusion matrix: 
 [[449 124]
 [  3  39]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.784 0.216]
 [0.071 0.929]]
Validation f1: 38.05
Validation accuracy: 79.35
Validation recall: 92.86
Validation precision: 23.93
Validation AUC: 0.5107751787808132
Validation class accuracies: [0.7835951134380453, 0.9285714285714286]
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 5 epochs
Epoch 12/65
154/154 - 235s - loss: 0.6428 - accuracy: 0.8816 - val_loss: 0.6791 - val_accuracy: 0.7138
Confusion matrix: 
 [[400 173]
 [  3  39]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.698 0.302]
 [0.071 0.929]]
Validation f1: 30.71
Validation accuracy: 71.38
Validation recall: 92.86
Validation precision: 18.40
Validation AUC: 0.4777291204319073
Validation class accuracies: [0.6980802792321117, 0.9285714285714286]
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 6 epochs
Lr has been adjusted to 0.0005000000237487257
Epoch 13/65
154/154 - 236s - loss: 0.6272 - accuracy: 0.9125 - val_loss: 0.6037 - val_accuracy: 0.8390
Confusion matrix: 
 [[484  89]
 [ 10  32]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.845 0.155]
 [0.238 0.762]]
Validation f1: 39.26
Validation accuracy: 83.90
Validation recall: 76.19
Validation precision: 26.45
Validation AUC: 0.4359643564430104
Validation class accuracies: [0.8446771378708552, 0.7619047619047619]
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 7 epochs
Epoch 14/65
154/154 - 235s - loss: 0.6223 - accuracy: 0.9060 - val_loss: 0.5833 - val_accuracy: 0.8715
Confusion matrix: 
 [[509  64]
 [ 15  27]]
/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Normalized Confusion matrix: 
 [[0.888 0.112]
 [0.357 0.643]]
Validation f1: 40.60
Validation accuracy: 87.15
Validation recall: 64.29
Validation precision: 29.67
Validation AUC: 0.5786709326129253
Validation class accuracies: [0.8883071553228621, 0.6428571428571429]
2021-06-23 00:10:00.135271: W tensorflow/core/framework/op_kernel.cc:1763] OP_REQUIRES failed at whole_file_read_ops.cc:116 : Not found: ../../data/txt_datasets/all_sw_coch_preprocessed_v2_param_v44_augm_v0_cleaned_8000/icbhi_BP18_pneumonia,C,P R U,57,M_0.txt; No such file or directory
The validation tracker metric at 0.44970414201183434 hasn't increased by 0 in 8 epochs
Epoch 15/65
Traceback (most recent call last):
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/eager/context.py", line 2113, in execution_mode
    yield
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/iterator_ops.py", line 733, in _next_internal
    output_shapes=self._flat_output_shapes)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/ops/gen_dataset_ops.py", line 2579, in iterator_get_next
    _ops.raise_from_not_ok_status(e, name)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/framework/ops.py", line 6862, in raise_from_not_ok_status
    six.raise_from(core._status_to_exception(e.code, message), None)
  File "<string>", line 3, in raise_from
tensorflow.python.framework.errors_impl.NotFoundError: ../../data/txt_datasets/all_sw_coch_preprocessed_v2_param_v44_augm_v0_cleaned_8000/icbhi_BP18_pneumonia,C,P R U,57,M_0.txt; No such file or directory
	 [[{{node ReadFile}}]] [Op:IteratorGetNext]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/alirachidi/classification_algorithm/trainers/main/models/conv/train2240.py", line 412, in <module>
    train_model(train_file, job_dir, params, wav_params, spec_params, model9, job_count=0)
  File "/home/alirachidi/classification_algorithm/trainers/main/models/conv/train2240.py", line 355, in train_model
    callbacks=[metrics_callback]
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/wandb/integration/keras/keras.py", line 124, in new_v2
    return old_v2(*args, **kwargs)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 1141, in fit
    return_dict=True)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 1389, in evaluate
    tmp_logs = self.test_function(iterator)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 1233, in test_function
    return step_function(self, iterator)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/keras/engine/training.py", line 1223, in step_function
    data = next(iterator)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/iterator_ops.py", line 747, in __next__
    return self._next_internal()
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/data/ops/iterator_ops.py", line 739, in _next_internal
    return structure.from_compatible_tensor_list(self._element_spec, ret)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/eager/context.py", line 2116, in execution_mode
    executor_new.wait()
  File "/home/alirachidi/anaconda3/envs/LungSoundClass/lib/python3.7/site-packages/tensorflow/python/eager/executor.py", line 69, in wait
    pywrap_tfe.TFE_ExecutorWaitForAllPendingNodes(self._handle)
tensorflow.python.framework.errors_impl.NotFoundError: ../../data/txt_datasets/all_sw_coch_preprocessed_v2_param_v44_augm_v0_cleaned_8000/icbhi_BP18_pneumonia,C,P R U,57,M_0.txt; No such file or directory
	 [[{{node ReadFile}}]]
wandb: Waiting for W&B process to finish, PID 3142
wandb: Program failed with code 1. 
wandb: Find user logs for this run at: /home/alirachidi/classification_algorithm/trainers/main/wandb/offline-run-20210622_230757-3bhij3ux/logs/debug.log
wandb: Find internal logs for this run at: /home/alirachidi/classification_algorithm/trainers/main/wandb/offline-run-20210622_230757-3bhij3ux/logs/debug-internal.log
wandb: Run summary:
wandb:   train_accuracy 0.90598
wandb:     val_accuracy 0.87154
wandb:         _runtime 3498
wandb:       _timestamp 1624406776
wandb:            _step 106
wandb:       train_loss 0.62233
wandb:         val_loss 0.5833
wandb:       val_recall 0.64286
wandb:    val_precision 0.2967
wandb:           val_f1 0.40602
wandb:          val_auc 0.57867
wandb:               lr 0.0005
wandb: Run history:
wandb:   train_accuracy ▁▅▅▆▆▆▆▆▇▇▇▇██
wandb:     val_accuracy ███▅▄▇▁▆▆▂▆▄▆▇
wandb:         _runtime ▁▁▁▂▂▂▂▂▃▃▃▃▃▃▃▃▄▄▄▄▄▄▅▅▅▅▅▆▆▆▆▆▇▇▇▇▇███
wandb:       _timestamp ▁▁▁▂▂▂▂▂▃▃▃▃▃▃▃▃▄▄▄▄▄▄▅▅▅▅▅▆▆▆▆▆▇▇▇▇▇███
wandb:            _step ▁▁▁▂▂▂▂▂▂▃▃▃▃▃▄▄▄▄▄▄▅▅▅▅▅▅▆▆▆▆▆▇▇▇▇▇▇███
wandb:       train_loss █▂▂▂▂▂▂▂▁▁▁▁▁▁
wandb:         val_loss ▄▁▁▃▅▂█▃▃▅▃▃▂▂
wandb:       val_recall ▁▁▁▆█▇██▇███▆▆
wandb:    val_precision ▁▁▁▅▅█▄▆▇▄▇▅▇█
wandb:           val_f1 ▁▁▁▅▆█▄▇▇▅▇▆▇▇
wandb:          val_auc ▁▁▁▅▅▇▅▆▆█▇▆▆█
wandb:               lr ████████████▁▁
wandb: 
wandb: You can sync this run to the cloud by running:
wandb: wandb sync /home/alirachidi/classification_algorithm/trainers/main/wandb/offline-run-20210622_230757-3bhij3ux


 See logs at ../../new_cache/conv__2240__06_2307__v39__augm/logs/